psi4/driver/procrouting/proc.py

```diff
@@ -3464,21 +3465,38 @@ def format_vector(label, data):
         lines += [""]
         core.print_out("\n".join(lines) + "\n")
 
+    gauge = core.get_option("ADC", "GAUGE").lower()
+    if gauge == "velocity":
+        gauge_short = "VEL"
+    elif gauge == "length":
+        gauge_short = "LEN"
+    else:
+        raise ValidationError(f"Gauge {gauge} not recognised for ADC calculations.")
+
     computed = {}
     if any(prop in properties for prop in ("TRANSITION_DIPOLE", "OSCILLATOR_STRENGTH")):
         data = state.transition_dipole_moments
         computed["Transition dipole moment (in a.u.)"] = data
-        adc_wfn.set_variable(name + " transition dipoles", core.Matrix.from_array(data))
+        adc_wfn.set_variable(f"{name} transition dipoles", core.Matrix.from_array(data))
 
     if "OSCILLATOR_STRENGTH" in properties:
-        data = state.oscillator_strengths.reshape(-1, 1)
-        computed["Oscillator strength (length gauge)"] = data
-        adc_wfn.set_variable(name + " oscillator strengths", core.Matrix.from_array(data))
+        if gauge == "velocity":
+            data = state.oscillator_strengths_velocity.reshape(-1, 1)
+        else:
+            data = state.oscillator_strengths.reshape(-1, 1)
+        computed[f"Oscillator strength ({gauge} gauge)"] = data
+        adc_wfn.set_variable(f"{name} oscillator strengths ({gauge_short})",
+                             core.Matrix.from_array(data))
+
+    if "ROTATIONAL_STRENGTH" in properties:
+        data = state.rotatory_strengths.reshape(-1, 1)
+        computed["Rotational strength"] = data
+        adc_wfn.set_variable(f"{name} rotational strengths", core.Matrix.from_array(data))
```

Comment:
Could you also add the gauge to the rotational strength variable? To have consistency. Whatever is used in adcc.